### DISCUSSION TITLE
Vomiting animation desync in multiplayer

### DISCUSSION THREAD

**Discussion by A2ur3** [FakeFishGames/Barotrauma#15063] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

https://github.com/user-attachments/assets/1ef84c5f-a68e-4ea7-9749-b452c2e1c017

### Reproduction steps

1. start a multiplayer
2. giveaffliction nausea 100
3. run around and wait for vomiting

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.
